index.js

```diff
@@ -1,3 +1,7 @@
-/* for module loading using webpack or similar package bundlers */
-window.Flow = require('./dist/ng-flow-standalone');
-module.exports = 'flow';
+/* backwards compatable with deprecated index.js*/
```

Comment:
this file is ignored by commonjs loaders because of the 'main' key in package.json so I log a warning.